tests/ci/pr_info.py

```diff
@@ -288,6 +289,11 @@ def __init__(
     def is_master(self) -> bool:
         return self.number == 0 and self.head_ref == "master"
 
+    def is_release(self) -> bool:
+        return self.number == 0 and bool(
+            re.match(r"^2[1-9]\.[1-9][0-9]*$", self.head_ref)
```

Comment:
Let's use `'release' in self.labels`. It will cover everything, including cloud releases

Reply:
to be done